src/outpack/search.py

```diff
@@ -70,6 +71,8 @@ def as_query(query: Union[Query, str]) -> Query:
     if isinstance(query, Query):
         return query
     else:
+        if is_outpack_id(query):
+            query = f"'{query}'"
```

Comment:
Is this something we want to add natively to the outpack parser?

Reply:
Yeah I think we should do, I've made a ticket https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5103/Search-for-packet-with-ID-without-quotes